src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -1376,4 +1376,81 @@ static public int endOfShard(int shardIndex, int shardSize) {
     public enum IntervalBreakpointType {
         START_BREAKPOINT, END_BREAKPOINT
     }
+
+    /**
+     * Creates a {@link SimpleInterval} on a coordinate from the provided sequence dictionary.
+     *
+     * @param dictionary reference dictionary.
+     * @param contig     the new location contig name.
+     * @param start      the new location start base index.
+     * @param stop       the new location stop base index.
+     *
+     * @return never {@code null}.
+     * @throws UserException if there was some problem when creating the location.
+     */
+    public static SimpleInterval createInterval(final SAMSequenceDictionary dictionary, final String contig, final int start, final int stop) {
+        Utils.nonNull(dictionary, "null dictionary");
+        Utils.nonNull(contig, "null contig");
+        if (!SimpleInterval.isValid(contig, start, stop)) {
+            throw new UserException(String.format("Invalid coordinate: %s:%s-%s", contig, start, stop));
+        }
+        final int contigIndex = dictionary.getSequenceIndex(contig);
+        if (contigIndex == -1) {
+            throw new UserException(String.format("Invalid coordinate: %s:%s-%s (contig not in dictionary)", contig, start, stop));
```

Comment:
I would personally probably change these to `IllegalArgumentException`, in this class most of the `UserExceptions` seem to be related invalid input files which these methods don't deal with directly. These errors look to mostly be relevant for test files anyway.

Reply:
It is also if the codepath can be related with an user invalid argument or configuration, but I understand that in this case it can be different. I've done it like this because in the previous class it was expected to throw `UserException`...

Reply:
Done. Changed everything to `IllegalArgumentException`